samples/sampler/stories/default/Marquee.js

```diff
@@ -6,55 +8,64 @@ import {storiesOf} from '@storybook/react';
 import {I18nContextDecorator} from '@enact/i18n/I18nDecorator';
 
-const I18nMarqueeBase = ({rtl}) => {
-	// fn to parse the padding value which is invoked later to keep the knob ordered
-	const spacing = () => {
-		const value = text('marqueeSpacing', Marquee, '50%');
-		if (value && value.indexOf('%') > 0) {
-			return value;
-		}
-
-		return Number.parseInt(value);
-	};
+Marquee.displayName = 'Marquee';
+const Config = mergeComponentMetadata('Marquee', Marquee);
 
-	const marqueeText = rtl ? 'قفز الثعلب البني السريع فوق الكلب الكسول. طائر الفاصوليا يطير عند غروب الشمس.' :
-		'The quick brown fox jumped over the lazy dog. The bean bird flies at sundown.';
+const I18nMarqueeBase = kind({
+	name: 'I18nMarquee',
 
-	const disabled = boolean('disabled', Marquee);
+	propTypes: {
+		rtl: PropTypes.bool
+	},
 
-	return (
-		<section>
+	render: ({rtl, ...rest}) => {
+		const marqueeText = rtl ? 'قفز الثعلب البني السريع فوق الكلب الكسول. طائر الفاصوليا يطير عند غروب الشمس.' : 'The quick brown fox jumped over the lazy dog. The bean bird flies at sundown.';
+		return (
 			<Marquee
-				alignment={select('alignment', [null, 'left', 'right', 'center'], Marquee)}
-				disabled={disabled}
-				forceDirection={select('forceDirection', [null, 'rtl', 'ltr'], Marquee)}
-				marqueeDelay={number('marqueeDelay', Marquee, 1000)}
-				marqueeDisabled={boolean('marqueeDisabled', Marquee)}
-				marqueeOn={select('marqueeOn', ['hover', 'render'], Marquee, 'render')}
-				marqueeOnRenderDelay={1000}
-				marqueeResetDelay={number('marqueeResetDelay', Marquee, 1000)}
-				marqueeSpacing={spacing()}
-				marqueeSpeed={number('marqueeSpeed', Marquee, 60)}
-				style={{width: '600px'}}
+				{...rest}
 			>
 				{marqueeText}
 			</Marquee>
-			{disabled ? <p style={{fontSize: '70%', fontStyle: 'italic'}}><sup>*</sup>Marquee does not visually respond to <code>disabled</code> state.</p> : <p />}
-		</section>
-	);
-};
-
-I18nMarqueeBase.propTypes = {
-	rtl: PropTypes.bool
-};
+		);
+	}
+});
 
 const I18nMarquee = I18nContextDecorator({rtlProp: 'rtl'}, I18nMarqueeBase);
 
 storiesOf('Agate', module)
 	.add(
 		'Marquee',
-		() => (<I18nMarquee />),
+		() => {
+			const value = text('marqueeSpacing', Config, '50%');
+			const spacing = () => {
+				if (value && value.indexOf('%') > 0) {
+					return value;
+				}
+
+				return Number.parseInt(value);
+			};
+
+			const disabled = boolean('disabled', Config);
+			return (
+				<section>
+					<I18nMarquee
+						disabled={disabled}
+						alignment={select('alignment', [null, 'left', 'right', 'center'], Config)}
+						forceDirection={select('forceDirection', [null, 'rtl', 'ltr'], Config)}
+						marqueeDelay={number('marqueeDelay', Config, 1000)}
+						marqueeDisabled={boolean('marqueeDisabled', Config)}
+						marqueeOn={select('marqueeOn', ['hover', 'render'], Config, 'render')}
+						marqueeOnRenderDelay={1000}
+						marqueeResetDelay={number('marqueeResetDelay', Config, 1000)}
+						marqueeSpacing={spacing()}
```

Comment:
To make it appear in alphabetical order, I suggested changing it like this:
marqueeSpacing={spacing(text('marqueeSpacing', Config, '50%'))}